app/assets/javascripts/components/StickyColumn.js

```diff
@@ -0,0 +1,215 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var StickyColumn,
+      defaultConfig = {},
+      $parent,
+      contentHeight,
+      topMargin,
+      top,
+      bottom,
+      isFixed,
+      isAtBottom,
+      isInSidebar,
+      mainContentSelector = '.l-article-3col-main',
+      parentSelector = '.l-article-3col-right',
+      fixedClass = 'related-links--desktop-fixed',
+      bottomClass = 'related-links--desktop-bottom';
+
+  StickyColumn = function($el, config) {
```

Comment:
Will there be any JS tests for this? As it could be useful for Dough in the future.